src/components/templates/PartyMap/components/Map/Map.module.scss

```diff
@@ -13,12 +13,30 @@
   // Needed to override inline styles that are only used on larger screens
   background-size: cover !important;
 
-  @media (max-width: calc(#{breakpoint.$md} - 1px)) {
-    z-index: -1;
+  @media (min-width: breakpoint.$md) {
+    position: absolute;
   }
+}
+
+.spaceName {
+  position: relative;
+  width: 100%;
+  height: 66vh;
+  box-sizing: border-box;
+  padding-top: 54px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 34vw;
+  letter-spacing: -0.07em;
+  line-height: 0.8;
```

Comment:
[Suggestion] These hard coded values look like they would benefit if they were transformed into constants

Reply:
moved to variables: https://github.com/sparkletown/sparkle/pull/3071/commits/53ad693cb533a0eaa1fddc004d2ab374630c2be1